index.test-d.ts

```diff
@@ -0,0 +1,43 @@
+import { expectType } from 'tsd';
```

Comment:
Can this be moved to the `test` folder?

Reply:
I wonder if it is better to move both `index.d.ts` and `index.test-d.ts` in a new `types` folder.